src/plugins/resolver/resolver.c

```diff
@@ -47,10 +47,15 @@
 #endif
 
 #ifdef ELEKTRA_LOCK_MUTEX
-#ifndef PTHREAD_RECURSIVE_MUTEX_INITIALIZER_NP
+#if defined(PTHREAD_RECURSIVE_MUTEX_INITIALIZER_NP)
+static pthread_mutex_t elektra_resolver_mutex = PTHREAD_RECURSIVE_MUTEX_INITIALIZER_NP;
```

Comment:
It should be camel case (elektraResolverMutex). Rename preferable also in a separate commit.